.github/workflows/locize-pull-published-sync-pr.yml

```
@@ -0,0 +1,53 @@
+      - name: Display Dispatch Payload
+        run: |
+          echo "Received Version: ${{ github.event.client_payload.payload.version }}"
+          echo "Full Payload: ${{ toJson(github.event.client_payload) }}"
+      - name: Download Translations from locize
+        uses: locize/download@v1
+        with:
+          project-id: ${{ secrets.LOCIZE_PROJECT_ID }}
+          version: ${{ github.event.client_payload.payload.version }}
+          path: "client/src/locales"
+          path-mask: "{{language}}/{{namespace}}"
+          clean: true # Ensures old translation files are removed before downloading new ones
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)